### ISSUE TITLE
Python client randomly (rarely) return None as field value, while cmd line client and WebUI returns correct value

### ISSUE THREAD

**Issue by rusnackor** [influxdata/influxdb-client-python#662]
### Specifications

* Client Version: influxdb-client-python/1.40.0
* InfluxDB Version: 2.7.3
* Platform: alpine-linux

### Code sample to reproduce problem

```python
result_data = self._client.query_api().query(query, params=query_parameters)
```

### Expected behavior

I cannot give you details publicly (but I have permission to share what you need privately).
So, I will explain generically what is the case.

We are storing data in 4 measurements, inside one bucket.
We store quite a lot of data, few hundrets per second.
While write flow keeps running (writing), we are also querying same measurements for the recently appeared data and forwarding them to other streams.

Right now, we are doing kind of stress test, so we have quite a lot of data flowing to and from DB.

Sometimes (what causes it is unknown for us) we are having strange behavior when we are querying data from InfluxDB using `python-client`: one of the fields has `None` as return value.
When we later run same query in WebUI or command line client, there is a value in that field, its **not `None`**.

_**Lets say, the field I am talking about is called `test_id`. I will put some examples:**_

This is expected behavior, that happens like 99,9% of times: we write `test_id` as some `integer`, lets say `1026`:
```python
Point(measurement).time(acq_ts).tag("status", 4).field("test_id", 1026).field("value", 0))
```
when we get query result, we get this (`'test_id': 1026`):
```python
FluxRecord() table: 176, {'result': '_result', 'table': 176, '_time': datetime.datetime(2024, 7, 1, 18, 17, 6, 954000, tzinfo=datetime.timezone.utc), '_measurement': 'test-measurement', 'test_id': 1026, 'status': 4, 'value': 0}
```
this is completely fine, thats what I want to get.

### Actual behavior

However, sometimes (like ~5 times a day) we have this:
**When this bug happens, we write `test_id` as some integer, lets say `1026`:**
```python
Point(measurement).time(acq_ts).tag("status", 4).field("test_id", 1026).field("value", 0))
```
but when we get query result, we get this (`'test_id': None`):
```python
FluxRecord() table: 176, {'result': '_result', 'table': 176, '_time': datetime.datetime(2024, 7, 1, 18, 17, 6, 954000, tzinfo=datetime.timezone.utc), '_measurement': 'test-measurement', 'test_id': None, 'status': 4, 'value': 0}
```
later if I run same query in WebUI or command line client, there is a value for the `test_id` field, and its value is `1026`.
Thats **not correct behavior.**

### Additional info

What I have noticed (and might give you some clues) is, that if we write `None` value in the `field`. When we query data afterwards, that `field` is not even mention in `FluxRecord()` object, there is no `key` with that `field` name in `values` dictionary.
However, in this case, there is `key` with that `field` name in `FluxRecord()` object, but its `value` is `None`.

**In case, we write `test_id` as `None`:**
```python
Point(measurement).time(acq_ts).tag("status", 4).field("test_id", None).field("value", 0))
```
What we will get as query result is:
```python
FluxRecord() table: 176, {'result': '_result', 'table': 176, '_time': datetime.datetime(2024, 7, 1, 18, 17, 6, 954000, tzinfo=datetime.timezone.utc), '_measurement': 'test-measurement', 'status': 4, 'value': 0}
```
and that is also **completely ok behaviour**.

**Comment by bednar** [comment]
Hi @rusnackor,

Thank you for using our client.

Do you use an aggregation query? It may be causing this behaviour. Could you please share the query you are executing against InfluxDB?

Additionally, could you enable the debug output on the client by setting the `debug=True` parameter? This will help us trace which response is causing the error.

> What I have noticed (and might give you some clues) is, that if we write None value in the field. When we query data afterwards, that field is not even mention in FluxRecord() object, there is no key with that field name in values dictionary.

Regarding your observation about writing `None` values: if a field’s value is `None`, it is not stored in InfluxDB. Consequently, it won’t appear in the FluxRecord() object; the corresponding key will be absent from the values dictionary.

Best Regards

**Comment by rusnackor** [comment]
Hello.

I am not using any agregation, my query is:
```python
f'from(bucket: "{self.bucket_name}")'
f' |> range(start: _start)'
f' |> filter(fn: (r) => r["_measurement"] == "measurement_name")'
f' |> drop(columns: ["_start", "_stop"])'
f' |> pivot(rowKey: ["_time"], columnKey: ["_field"], valueColumn: "_value")'
f' |> filter(fn: (r) => r["row_id"] > {id_start})'
 ```
 
In the measurement, I have 5 `tag` and 7 `field` columns. What I have observed is, that a `tag` column is never `None` it is allways `field`. And also, that it is random, its not allways same `field`, number of `None` fields is random, and also is random which onea are `None`.
 
 Another observation is, that those `None` values occur in DataPoints, which were very recently written (like few microseconds time difference between write and read operations). So, what I am consaidering is, that it might be possible that I am reading them before InfluxDB fully process the data. Is it even possible? I was under impression that databases should have sorted this out :)
 
BUT than again, the InfluxDB is not classic realtion DB, and if you do not use `pivot` in the query, you get each `field` value with its `tag` values in separate tables. So maybe Influx internally does not have relations between them and is writting them separately, and that might cause this issue when reading right after write (or at the same time).
 
 I have changed my testbed little bit, added 1 min delay between write and read. Will let you know how my experiment with delayed reading resulted.
